xcell/mappers/mapper_dummy.py

```diff
@@ -189,6 +190,15 @@ def get_cl_coupled(self):
             self.cl_coupled += self.custom_offset
         return self.cl_coupled
 
+    def get_cls_covar_coupled(self):
+        if self.cls_cov is None:
+            clc = self.get_cl_coupled()
+            self.cls_cov = {'cross': clc,
+                            'auto_11': clc,
```

Comment:
Wouldn't it be better that each of these were different? My point is that if 'auto' is not True in the `ClFid` (L. 192), then the `cl_cov_cp` will be the same. It might make the tests a bit confusing.

Reply:
OK, I've subtracted `custom_offset` from `cross` and `auto_12` so we can test that they are different.